docs/source/index.rst

```diff
@@ -1,16 +1,27 @@
 .. image:: ../Pseudopeople-logo_FINAL_2023.04.11_psdppl-logo_blue-ombre.png
 
+.. _IHME: https://www.healthdata.org/
+.. _Vivarium: https://vivarium.readthedocs.io/en/latest/
+.. _FEBRL: http://users.cecs.anu.edu.au/~Peter.Christen/Febrl/febrl-0.3/febrldoc-0.3/manual.html
+.. _GeCo: https://dl.acm.org/doi/10.1145/2505515.2508207
+.. _SOG: https://web.archive.org/web/20170830050229/http:/mitiq.mit.edu/ICIQ/Documents/IQ%20Conference%202009/Papers/3-B.pdf
+
 pseudopeople is a Python package that generates realistic simulated data about a
-fictional United States population over multiple decades, for use in testing entity resolution (record linkage)
-methods.
+fictional United States population, designed for use in testing entity
+resolution (record linkage) methods or other data science algorithms at scale.
 
 | 🙈 **Simulated**: These are made-up people! No need to worry about confidentiality.
 | 📝 **Versatile**: Generate multiple datasets about the same population: censuses, surveys, and administrative records.
 | ✔️ **Verifiable**: Ground-truth unique identifiers are present in every dataset for checking link correctness.
 | ⚙️ **Customizable**: Configure the levels of noise in each dataset.
 | 💪 **Full-scale**: Supports generating datasets at the size of the real-life US population.
 
-Pseudopeople is currently in a public beta release.
+The fictional US population was generated by stochastically simulating multiple
+decades of population dynamics such as fertility, mortality, migration and
+employment. pseudopeople builds on this fictional population data by simulating
+errors in the data collection process to create realistic, noisy datasets.
```

Comment:
We could link this to the vivarium-research repo and/or the vivarium_prl_synthpop repo for readers who want more details, but maybe that is TMI.  When we have a "data note" that summarizes the concept model doc from the vivarium-research repo, we should link to it here.